src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -839,11 +850,11 @@ public ImmutableArray<T> RemoveRange(ImmutableArray<T> items)
         public ImmutableArray<T> RemoveRange(ImmutableArray<T> items, IEqualityComparer<T> equalityComparer)
         {
             var self = this;
+            self.ThrowNullRefIfNotInitialized();
```

Comment:
@James-Ko, it may very well have been a mistake, and it may very well make things more consistent with the rest of the system... I can appreciate both of those things.  It's just that at this point the library has already shipped, so we need to be cognizant of any changes in behavior and either avoid them or at least be very thoughtful about when and why they're made.  Such changes should not be made as part of a PR focused on performance improvements.  Thanks for understanding.